torch/csrc/jit/python/script_init.cpp

```diff
@@ -987,14 +988,31 @@ void initJitScriptBindings(PyObject* module) {
             pyIValueDeepcopy(IValue(self._ivalue()), memo).toObject());
       });
 
-  // Used by torch.Package to save TS objects in unified format
+  // Used by torch.package to save ScriptModule objects in unified format and
+  // to coordinate sharing of storages between eager and ScriptModules with
+  // ScriptModuleSerializer's inner SerializationStorageContext.
   py::class_<ScriptModuleSerializer>(m, "ScriptModuleSerializer")
       .def(py::init<caffe2::serialize::PyTorchStreamWriter&>())
       .def("serialize", &ScriptModuleSerializer::serialize_unified_format)
       .def(
           "write_files",
           &ScriptModuleSerializer::writeFiles,
-          py::arg("code_dir") = ".data/ts_code/code/");
+          py::arg("code_dir") = ".data/ts_code/code/")
```

Comment:
Let's create a separate binding for `SerializationStorageContext`, for two reasons:
1. It's more readable/understandable for there to be a 1:1 mapping of pybind classes to cpp classes, with the same API exposed.
2. It currently gives the false impression that we only need `SerializationStorageContext` in the context of ScriptModule serialization.